resources/dicts/patrols/general/border.json

```diff
@@ -1,1292 +1,248 @@
-                "weight": 10,
-                "injury": [
-                    {
-                        "cats": ["r_c"],
-                        "injuries": ["small_bite_injury"],
-                        "scars": ["LEGBITE"]
-                    }
-                ],
-                "history_text": {
-                    "scar": "This cat got a scar from a fight with a small dog, but {PRONOUN/m_c/subject} always {VERB/m_c/say/says} it was a huge dog."
-                }
+                "weight": 20
             }
         ]
     },
     {
-        "patrol_id": "gen_bord_smalldog3",
+        "patrol_id": "Gen_bord_balancing_tracks",
         "biome": ["Any"],
         "season": ["Any"],
         "types": ["border"],
         "tags": [],
-        "patrol_art": "bord_general_intro",
-        "min_cats": 4,
+        "patrol_art": "gen_bord_trackrogue",
+        "min_cats": 1,
         "max_cats": 6,
         "min_max_status": {},
         "weight": 5,
-        "intro_text": "Your patrol comes across a small dog, right on the border of c_n territory.",
-        "decline_text": "p_l decides not to pursue the dog, instead cutting the patrol short to head back to camp and report it.",
-        "chance_of_success": 10,
+        "intro_text": "p_l heads out to patrol c_n's borders, but spots some tracks.",
+        "decline_text": "p_l decides to not investigate the tracks.",
+        "chance_of_success": 50,
         "success_outcomes": [
             {
-                "text": "The patrol paces towards the small dog, claws out, fur up, backed up with the courage of their Clanmates. The dog breaks into a barking fit, but that's their opportunity to attack, covering each other as they push the creature away from their lands.",
-                "exp": 10,
-                "weight": 20,
-                "relationships": [
-                    {
-                        "cats_to": ["patrol"],
-                        "cats_from": ["patrol"],
-                        "mutual": false,
-                        "values": ["respect", "comfort", "trust"],
-                        "amount": 5
-                    }
-                ]
+                "text": "p_l identifys the tracks as belonging to some prey.",
+                "exp": 5,
+                "weight": 20
             },
             {
-                "text": "Mobbed by cats and near blind with the blood in its eyes, the dog flees from c_n, yelping pitifully.",
-                "exp": 10,
-                "weight": 5,
-                "relationships": [
-                    {
-                        "cats_to": ["patrol"],
-                        "cats_from": ["patrol"],
-                        "mutual": false,
-                        "values": ["respect", "comfort", "trust"],
-                        "amount": 5
-                    }
-                ]
+                "text": "The tracks lead p_l to a nearby red berry bush. p_l moves on, noting it down to inform the medicine cat.",
```

Comment:
kinda love the vague implication it could be a deathberry bush ngl

Reply:
That was the plan yeah! Who *knows* what it is, the warrior doesn't, so best just to let the healer know XD